test/types/index.ts

```diff
@@ -417,7 +417,7 @@ server.setErrorHandler((err, request, reply) => {
 server.listen(3000, err => {
   if (err) throw err
   const address = server.server.address()
-  if (typeof address === 'object') {
+  if (typeof address === 'object' && address !== null) {
```

Comment:
Why this change? I think if it is null, the check will fail here anyways, right?

Reply:
typeof null returns 'object'

Reply:
@smartiniOnGitHub i think a better check here would be to use short-circuiting:
```js
if (address && typeof address === 'object')
```

Reply:
TIL. Thanks @Ethan-Arrowood !

Reply:
Hi, without that change the test fails; but not sure if this is the right fix ...

Reply:
@Ethan-Arrowood Changes applied, all tests good even now. Thanks for the help :-)